NBitcoin/RPC/RPCClient.Wallet.cs

```diff
@@ -209,14 +221,14 @@ public RPCClient LoadWallet(string filename, bool? loadOnStartup = null)
 			return GetWallet(result.Result.Value<string>("name"));
 		}
 
-		public async Task UnloadAsync(bool? loadOnStartup = null)
+		public async Task UnloadWalletAsync(string filename, bool? loadOnStartup = null)
 		{
-			var result = await SendCommandAsync(RPCOperations.loadwallet, loadOnStartup).ConfigureAwait(false);
+			var result = await SendCommandAsync(RPCOperations.loadwallet, filename, loadOnStartup).ConfigureAwait(false);
```

Comment:
If `loadOnStartup` is `null`, then the best is to call `SendCommandAsync` without it, leaving the default to bitcoin core rather than passing null.

We need to do this to protect against breaking change which might come from bitcoin core. (and also increase compatiblity of RPC Client with other shitcoins)

Reply:
I noticed I did not do it for the other methods. Will do.